kleva-j/iReporter
# Feature/#163468248/create a record from the frontend

#### What does this PR do?
- Ensures a user can create a new record from the frontend

#### Description of the task to be completed
 - create functions to make requests to the API from the frontend to create a record on the app

#### How should this be manually tested?
- Clone the repository
- Check out into the branch Feature/#163468248/create-a-record-from-the-frontend
- Install all dependencies
- run `npm start` to run the app successfully.

#### What are the relevant pivotal tracker stories?
[163468248](https://www.pivotaltracker.com/story/show/163468248)
